inventory/aws/ec2.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
ec2.py is an upstream ansible script. If we modify it, we have to make sure to always carry those modification forward. I'd prefer to not modify anything we get from upstream. What is the intention with specifying python2 here, is it just a 'best practice' or do we need this so that it functions correctly in certain configurations?